packages/components/rollup.config.js

```diff
@@ -34,10 +33,7 @@ export default [
                 extensions: ['.ts', '.tsx'],
             }),
             commonjs(),
-            ts({
-                transpiler: 'babel',
-                browserslist: false,
-            }),
+            typescript(),
```

Comment:
@crspeller Do you happen to remember why you used rollup-plugin-ts over `@rollup/plugin-typescript` before?

Reply:
I believe the `@rollup/plugin-typescript` one doesn't work with babel. Is there some motivation for changing it? I think having babel in the pipeline will make migrating components easier.

Reply:
Interesting. I would've thought babel would be its own explicit plugin rather than being implicitly included by another one.

I guess whether or not we need Babel depends on if we're ever including raw JS in this package or anything else that isn't covered by the TypeScript compiler. I haven't taken a look at Babel in a while to see what's actually still needed from it. I see that there's plugins for libraries (lodash and styled-components), some ES proposals that may have native support now, and something for hot loading. The lodash and styled-components ones at least seem like they'd be relevant in this package, so it's probably best to keep babel in the pipeline still.

@nevyangelova Is this change still needed now that we fixed the CircleCI issues? `@rollup/plugin-typescript` was involved there, so I wonder if we can revert this change